doc/colvars-refman-main.tex

```diff
@@ -6489,6 +6489,339 @@
+It does so by reweighting via kernel density estimation the unbiased distribution in the CV space, $P(\boldsymbol{\xi})$.
+A compression algorithm is used to prevent the number of kernels from growing linearly with the simulation time.
+The bias at step $n$ is
+\begin{equation}\label{eq:opes_potential}
+V_n(\boldsymbol{\xi}) = -k_{\mathrm{B}}T(1-1/\gamma)\log\left(\frac{P_n(\boldsymbol{\xi})}{Z_n}+\epsilon\right),
+\end{equation}
+where the probability $P_n (\boldsymbol{\xi})$ and the normlization factor $Z_n$ is computed as
+\begin{equation}
+P_n (\boldsymbol{\xi}) = \frac{\sum_k^n w_k G(\boldsymbol{\xi}, \boldsymbol{\xi}_k)}{\sum_k^n w_k}
+\end{equation}
+\begin{equation}
+Z_n = \frac{1}{|\Omega_n|}\int_{\Omega_n} P_n (\boldsymbol{\xi}) \mathrm{d}\boldsymbol{\xi},
+\end{equation}
+where the weights $w_k$ are given by $w_k = \exp(\beta V_{k-1}(\boldsymbol{\xi}_k))$, and the Gaussian kernels
+$G(\boldsymbol{\xi}, \boldsymbol{\xi}')=h\exp\left[-1/2(\boldsymbol{\xi}-\boldsymbol{\xi}')^T \Sigma^{-1}(\boldsymbol{\xi}-\boldsymbol{\xi}')\right]$ have a diagonal covariance matrix $\Sigma_{ij}=\sigma^2 \delta_{ij}$ and fixed height $h=\Pi_i (\sigma_i \sqrt{2\pi})^{-1}$ (see Ref.\cite{Invernizzi2020} for a complete description of the method). This section (aside from the "implementation notes") is largely based on the \href{https://www.plumed.org/doc-master/user-doc/html/_o_p_e_s__m_e_t_a_d__e_x_p_l_o_r_e.html}{PLUMED manual of the OPES module}.
+
+\cvsubsubsec{Implementation notes}{sec:colvarbias_opes_notes}
+The \texttt{opes{\textunderscore}metad} in Colvars is largely based on the code of the OPES module in PLUMED \cite{tribello_plumed_2014}, except that it has the following limitations:
+\begin{itemize}
+\item Colvars does not support chaining bias actions in PLUMED, so the Colvars OPES implementation cannot take the biasing energies of other biases, such as harmonic restraints, into account.
+\item Due to the same limitation of Colvars as descrbited above, you cannot collect samples on-the-fly by chaining \texttt{opes{\textunderscore}metad} with a weighted histogram for PMF calculations. Although the PMF calculation via reweighting is implemented into the \texttt{pmf} option, the functionality is quite limited. The CVs in \texttt{pmf{\textunderscore}colvars} must be among the biased ones. In other words, you cannot apply bias on $(\xi_1, \xi_2)$ and compute the PMF along $\xi_3$ by the on-the-fly reweighting using \texttt{pmf} (you can only compute the PMF along $\xi_1$, $\xi_2$ or $(\xi_1, \xi_2)$). In addition, you can only specify one set of CVs for estimating the PMF (you cannot apply the OPES bias on $(\xi_1, \xi_2, \xi_3)$ and compute two 2D PMFs along $(\xi_1, \xi_2)$ and $(\xi_2, \xi_3)$ simultaneously using the \texttt{pmf} option). However, you can always bypass this limitation by post-hoc reweighting with the trajectories.
```

Comment:
I guess that one need to use the `E_opes_metad1` column in the `colvars.traj` file to reweight? This may be made clear here.

Reply:
Done. See https://github.com/Colvars/colvars/pull/709/commits/8dae4f715398950a2462ebe55f6fed83f9acecf5#diff-a73ad1c57d1f15db8ebd3f50bf56f4f6a889d8c866aff3d3847536624be6ee42R6778